modules/claims_api/lib/claims_api/common/exceptions/json_schema_validation_error.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module Error
+    class JsonSchemaValidationError < StandardError
+      def initialize(error)
+        @title = 'Unprocessable Entity'
+        @source = error[:errors][0][:source]
```

Comment:
Do you mind making 'title' dynamic like source, etc?

Reply:
Ya good catch, updated